src/components/atoms/ScrollToTopButton/styles.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+import { ArrowUpIconStyleProps } from "./interface";
+import { ReactComponent as ArrowUp } from "../../../assets/img/icons/arrow-up.svg";
+import { minDevice } from "../../../assets/styles/breakpoints";
+
+const ScrollButton = styled.button`
+  position: fixed;
+  left: 85%;
+  top: 10%;
+  width: fit-content;
+  background: none;
+  border: none;
+  overflow-y: scroll;
+`;
+
+const ArrowUpIcon = styled(ArrowUp)<ArrowUpIconStyleProps>`
+  display: ${(props) => props.display};
```

Comment:
Is this needed to be configurable from the outside? 🤔

Reply:
This is this place when typescript was screaming at me :( We can discuss this during our daily :)